host/traceh.c

```diff
@@ -94,10 +116,11 @@ void initialize_log_config()
     char* env_log_all_streams = NULL;
     char* env_log_escape = NULL;
 
-    if (!_initialized)
+    if (!_initialized_log_config)
     {
         // inititalize if not already
-        _log_level = _env2log_level();
+        if (!_initialized_log_level)
```

Comment:
Removed the variable tracking _log_level.

oe_set_host_log_level() now calls initialize_log_config to avoid the scenario where if the application calls oe_set_host_log_level() before a call to OE_TRACE() occurs, then OE_TRACE() would over-write the _log_level set by the API oe_set_host_log_level().